services/core-web/src/components/mine/ProjectSummaries/MineProjectSummary.js

```diff
@@ -68,7 +68,4 @@ const mapDispatchToProps = (dispatch) =>
 
 MineProjectSummary.propTypes = propTypes;
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(AuthorizationGuard(Permission.IN_TESTING)(MineProjectSummary));
+export default connect(mapStateToProps, mapDispatchToProps)(MineProjectSummary);
```

Comment:
this should still be wrapped in the 'AuthorizationGuard' otherwise we'll be releasing when this pr gets merged haha